
---
bool success
string message
Vehicle[] vehicles